tools/autodeployment/builds.js

```diff
@@ -4,9 +4,10 @@ const mergeStream = require('merge-stream');
 const { ReReadable } = require('rereadable-stream');
 
 class Build {
-  constructor(type, githubData) {
+  constructor(type, githubData, sha) {
```

Comment:
Tested on staging and production.
For this to happen you need to add the sha as a parameter in lines 96 and 100. I think after that this is good to go

Reply:
I manually added the changes mentioned in the previous comment to staging and production and they're displaying the correct sha for both.

Reply:
Awesome, nice catch. Sorry I missed this.

I've added both, rebased, and pushed again.